src/plugins/curator/components/sortable-grid/ButtonToolbarAsset.jsx

```diff
@@ -65,12 +74,22 @@ export default class ButtonToolbarAsset extends React.Component {
           key={node.get('_id')}
           showEditSequence={showEditSequence}
         />
+        {!disabled && !isHovering && multiSelect && isSelected && !showEditSequence && (
+          <ButtonToolbar style={{ position: 'absolute', left: '0', top: '0', padding: '.5rem', width: '100%', justifyContent: 'space-between', minHeight: '44px' }}>
+            <>
+              <Checkbox checked={isSelected} onChange={() => onSelect(node)} />
+            </>
+          </ButtonToolbar>
+        )}
         {!disabled && isHovering && (
-          <ButtonToolbar style={{ position: 'absolute', right: '.5rem', top: '.5rem' }}>
+          <ButtonToolbar style={{ position: 'absolute', left: '0', top: '0', padding: '.5rem', width: '100%', justifyContent: 'space-between', minHeight: '44px' }}>
             {!showEditSequence && (
               <>
-                <ToolbarButton icon="edit" id="media-edit" tooltip="Edit" onMouseDown={() => onEditAsset(node)} />
-                <ToolbarButton icon="trash" id="media-remove" tooltip="Remove" onMouseDown={() => onRemoveAsset(node)} />
+                {multiSelect && <Checkbox checked={isSelected} onChange={() => onSelect(node)} />}
```

Comment:
@rsumilang is multiSelect ever false? is this prop  needed?

Reply:
There is nothing else that currently uses SortableGrid. However the component seems generic enough to display assets anywhere with a toolbar.  However I can imagine a thing that may want to display assets in the future in a grid but not require any functionality to individually select assets so I gave it an option.